spec

```diff
@@ -2736,8 +2736,11 @@ endseg
 beginseg
     name "ovl_En_Mb"
     include "build/src/overlays/actors/ovl_En_Mb/z_en_mb.o"
-    include "build/data/overlays/actors/z_en_mb.data.o"
+#ifdef NON_MATCHING
+	include "build/src/overlays/actors/ovl_En_Mb/ovl_En_Mb_reloc.o"
```

Comment:
Haven't reviewed the rest but just mentioning a pretty big problem with this PR.

It's a limitation of the way we current handle reloc, but we can't use auto generated reloc in files that don't have all functions under at least `NON_MATCHING`. And of course we can't have functions under `NON_MATCHING` if we don't use the generated reloc, since otherwise the build will be unstable.
So all this means we can't have `NON_MATCHING` functions in overlays that also have non decompiled or non equivalent functions, unfortunately.